core/static/styles/_forms.scss

```diff
@@ -0,0 +1,180 @@
+// GLOBAL FORM STYLES
+// ----------------------------
+//
+// REQUIRES...
+
+// From _colours.scss
+// $black
+// $white
+// $grey
+// $great-red
+// $mid-grey
+// $mid-blue
+// $focus-yellow
+
+// From _vars.scss
+// $spacing-3
+
+// From _typography.scss
+// @mixin body-s
+// @mixin body-m
+
+
+fieldset {
+  &:after {
+    clear: both;
+    content: "";
+    display: block;
+  }
+}
+
+input[type=text] {
+  @include body-m;
+  border: 2px solid $grey;
+  height: 50px;
+  display: block;
+  margin-bottom: $spacing-3;
+  padding: 0 5px;
+  text-transform: none;
+  width: 100%;
+}
+
+label {
+  @include body-m;
+  color: $grey;
+  display: inline-block;
+  
+  .description {
+    color: $mid-grey;
+    display: block;
+  }
+}
+
+select {
+  @include body-m;
+  display: block;
+  margin-bottom: $spacing-3;
+  border: 2px solid $grey;
+  padding: 10px;
+  background: $white url("/static/images/down-arrow.svg") no-repeat 98%/15px;
+  height: 50px;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  text-overflow: '';
+}
+
+textarea {
+  @include body-m;
+  border: 2px solid $grey;
+  display: block;
+  margin-bottom: $spacing-3;
+  padding: 5px;
+  width: 100%;
+}
+
+.error {
+  color: $great-red;
+}
+
+
+// Checkboxes + Radio buttons
+.checkbox,
+.radio {
+  @include body-m;
+  margin-bottom: $spacing-4;
+  position: relative;
+  padding: 4px 0 0 45px;
+  -ms-touch-action: manipulation;
+  touch-action: manipulation;
```

Comment:
ditto

Reply:
another css thing. specifically for touch screens https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action